bucketlist_app/helper.py

```diff
@@ -0,0 +1,49 @@
+def get_or_none(classmodel, **kwargs):
+    try:
+        return classmodel.objects.get(**kwargs)
+    except classmodel.DoesNotExist:
+        return None
+
+
+def pretty_date(time=False):
+    """
```

Comment:
Not sure if this comment style is fully PEP8 though

Reply:
Fixed.